### DISCUSSION TITLE
Collecting per second data

### DISCUSSION THREAD

**Discussion by Mufika** [jertel/vuegraf#138] [Q&A] [answered]
I want to collect per second data from an Emporia Vue device using the docker compose instance of Vuegraf.  I want to query the Emporia server every one or two hours for per seconds data. I'm setting up the following configuration variation as follows:
"updateIntervalSecs": 3600,
"detailedDataEnabled": "true",

Is this correct, i.e. polling per second data every hour?

**Comment by jertel** [comment, marked as answer]
You only need to specify the `detailedDataEnabled": true` setting. The intervals will default to what you stated above. It will collect per-second data every hour, and per-minute data every minute.
